taca/utils/bioinfo_tab.py

```diff
@@ -148,16 +148,20 @@ def get_ss_projects(run_dir):
         logger.error('Cannot find RunParameters.xml or runParameters.xml in the run folder for run {}'.format(run_dir))
         return []
     rp = RunParametersParser(os.path.join(run_dir, run_parameters_file))
-    try:
-        runtype = rp.data['RunParameters']['Setup']['Flowcell']
-    except KeyError:
-        logger.warn('Parsing runParameters to fetch instrument type, '
-                    'not found Flowcell information in it. Using ApplicationName')
-        try:
+    if 'Setup' in rp.data['RunParameters']:
```

Comment:
Based on what the last commit, wouldn't NextSeq be the else condition for this, like in the analysis.py file? Like this

`      if 'Setup' in rp.data['RunParameters']:`
       
        runtype = rp.data['RunParameters']['Setup'].get('Flowcell', '')
        if not runtype:
            logger.warn('Parsing runParameters to fetch instrument type, '
                        'not found Flowcell information in it. Using ApplicationName')
             runtype = rp.data['RunParameters']['Setup'].get('ApplicationName', '')
             if not runtype:
                logger.warn("Couldn't find runtype in 'ApplicationName', could be Novaseq. Trying 'Application'")
                runtype = rp.data['RunParameters']['Application']
    else:
        # This is the case for NextSeq 2000
        runtype = rp.data['RunParameters'].get('ApplicationName', '')
`

Reply:
Although, I'm not sure if the run parameters for Novaseq does have a field 'Setup'

Reply:
No, the run parameters for NovaSeq does not have the field "Setup"